src/main/java/org/infinispan/persistence/cloud/configuration/CloudStoreConfigurationParser70.java

```diff
@@ -0,0 +1,111 @@
+package org.infinispan.persistence.cloud.configuration;
+
+import static org.infinispan.commons.util.StringPropertyReplacer.replaceProperties;
+
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamException;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.PersistenceConfigurationBuilder;
+import org.infinispan.configuration.parsing.ConfigurationBuilderHolder;
+import org.infinispan.configuration.parsing.ConfigurationParser;
+import org.infinispan.configuration.parsing.Namespace;
+import org.infinispan.configuration.parsing.Namespaces;
+import org.infinispan.configuration.parsing.ParseUtils;
+import org.infinispan.configuration.parsing.Parser70;
+import org.infinispan.configuration.parsing.XMLExtendedStreamReader;
+import org.kohsuke.MetaInfServices;
+
+/**
+*
+* CloudStoreConfigurationParser70.
+* 
+* @since 7.0
+*/
+@MetaInfServices
+@Namespaces({
+      @Namespace(uri = "urn:infinispan:config:store:cloud:7.0",
+                 root = CloudStoreConfigurationParser70.ROOT_ELEMENT),
+})
+public class CloudStoreConfigurationParser70 implements ConfigurationParser {
```

Comment:
We don't need a 7.0 parser, make it 7.2 (as that's going to be our next release)